pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ProducerBuilder.java

```diff
@@ -343,6 +344,15 @@
      */
     ProducerBuilder<T> enableChunking(boolean enableChunking);
 
+    /**
+     * Max chunk message size in bytes. Producer chunks the message if chunking is enabled and message size is larger
+     * than max chunk-message size. By default, producer chunks based on max-message size configured at broker.
+     *
+     * @param chunkMaxMessageSize
+     * @return
+     */
```

Comment:
Which is the default value? Which value can be used to disable this feature? Can you please document it?

Reply:
added info.